apps/prairielearn/src/pages/instructorAssessmentDownloads/instructorAssessmentDownloads.ts

```diff
@@ -21,7 +34,73 @@ const sql = sqldb.loadSqlEquiv(import.meta.url);
 
 type Columns = [string, string][];
 
-function getFilenames(locals) {
+const AssessmentInstanceSubmissionRowSchema = z.object({
```

Comment:
I'd appreciate a close look at the two new schemas in the file, both this one and `ManualGradingSubmissionRowSchema`. I tried to base these directly on the underlying queries, but I might have missed the fact that something could be null.